src/main/assets/scss/main.scss

```diff
@@ -1,4 +1,5 @@
 @import '~govuk-frontend';
+@import 'accessible-autocomplete.min';
```

Comment:
This seemed to be causing an error for me - it couldn't find the accessible-autocomplete.min file.

~accessible-autocomplete seemed to work though. Not sure what could be causing it?